salt/modules/lxc.py

```diff
@@ -989,6 +1045,10 @@ def init(name,
     if users is None:
         users = []
     dusers = ['root']
+    # think to touch the default user generated by default ubuntu template which
+    # has a really unsecure password too ...
+    if __grains__['os'] in ['Ubuntu'] and 'ubuntu' not in users:
+        dusers.append('ubuntu')
```

Comment:
More over, can we prepare all the commits to push on the lxc front in this specific PR, or merge my stuff somewhere else as a start to new incoming corrections to make room for discussion but be consistent to the base code.
The burden to merge the different trees is really hard, i would love not to restart merges again & again, this is an useless stuff im redoing for now more than 6 monthes, i love contributing to salt, but please stop that !